app/src/main/java/org/jellyfin/androidtv/playback/MediaManager.java

```diff
@@ -227,8 +227,12 @@ public void onPlayerError(ExoPlaybackException error) {
                 mLibVLC = new LibVLC(TvApp.getApplication(), options);
 
                 mVlcPlayer = new org.videolan.libvlc.MediaPlayer(mLibVLC);
-                mVlcPlayer.setAudioOutput(Utils.downMixAudio() ? "opensles_android" : "android_audiotrack");
-                mVlcPlayer.setAudioOutputDevice("hdmi");
+                if(!Utils.downMixAudio()) {
+                    mVlcPlayer.setAudioDigitalOutputEnabled(true);
```

Comment:
It looks like this will returns a boolean indicating if it is successful or not, we should probably do something like:
```java
if (!Utils.downMixAudio() || !mVlcPlayer.setAudioDigitalOutputEnabled(true)) {
    mVlcPlayer.setAudioOutput("opensles_android");
}
```
So it will fallback to downmixing if it fails.

Reply:
Not an expert on VLC, but are we sure in the case that `setAudioDigitalOutputEnabled` fails, `opensles_android` is preferrable? FWIW audio was outputting previously WITHOUT `setAudioDigitalOutputEnabled` being called AND without downmixing, however it was not properly outputting Atmos.

Seems like if it is not always preferrable, might be less confusing for users to have to manually set this in preferences.

Reply:
Ah I think I follow now. This seems fine.

Should we be calling ‘mVlcPlayer.setAudioOutput("android_audiotrack");‘ here like you do below?